apps/transport/client/webpack.prod.js

```diff
@@ -6,6 +6,10 @@ console.log('webpack production configuration is used 🚀')
 
 module.exports = merge(common, {
     mode: 'production',
+    cache: {
```

Comment:
```suggestion
    # While this cache is applied during the actual production build, we rely on the fact
    # that folders are not cached over there, so that it does not have an impact there.
    cache: {
```

Reply:
(J'efface cette première suggestion qui ne me paraît pas optimale)